lib/src/references/pubspec_urls.dart

```diff
@@ -148,7 +148,8 @@ Future<UrlWithIssue> _checkUrl(
       Issue(
         "$name doesn't exist.",
         span: tryGetSpanFromYamlMap(pubspec.originalYaml, key),
-        suggestion: 'At the time of the analysis `$url` was unreachable.',
+        suggestion: 'At the time of the analysis `$url` was unreachable. '
+            'Make sure that the website is reachable via `HEAD` requests.',
```

Comment:
Can we point to documentation, perhaps:
```suggestion
            'Make sure that the website is reachable via [`HEAD`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods/HEAD) requests.',
```